gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -687,6 +691,12 @@ public GsonBuilder registerTypeAdapter(Type type, Object typeAdapter) {
     return this;
   }
 
+  private boolean isTypeObjectOrJsonElements(Type type) {
```

Comment:
Should probably be "JsonElement" (without "s") to match the class name:
```suggestion
  private boolean isTypeObjectOrJsonElement(Type type) {
```